src/main/java/uk/gov/beis/els/categories/washingmachines/controller/WashingMachinesController.java

```diff
@@ -83,31 +92,27 @@ public ModelAndView renderWasherDryer(@ModelAttribute("form") WasherDryerForm fo
 
   @PostMapping("/washer-dryer")
   @ResponseBody
-  public Object handleWasherDryerSubmit(@Validated({Default.class, PostMarch2021Field.class}) @ModelAttribute("form") WasherDryerForm form, BindingResult bindingResult) {
+  public Object handleWasherDryerSubmit(
+      @Validated({Default.class, PostMarch2021Field.class}) @ModelAttribute("form") WasherDryerForm form,
```

Comment:
I don't think the `PostMarch2021Field` validation group is needed here any more are you removed it from the form?